src/split-button.test.visuals.ts

```diff
@@ -130,6 +141,20 @@ for (const story of stories['Split Button']) {
             );
           });
 
+          test('<glide-core-split-button-primary-button>:focus', async ({
+            page,
+          }, test) => {
+            await page.goto(`?id=${story.id}&globals=theme:${theme}`);
+
+            await page
+              .locator('glide-core-split-button-primary-button')
```

Comment:
A case where a locator instead of a role selector will suffice because Split Button Primary Button is both unique and not an implementation detail of Split Button.